packages/database/supabase/schemas/space.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE IF NOT EXISTS public."DiscoursePlatform" (
+    id bigint DEFAULT nextval(
+        'public."entity_id_seq"'::regclass
+    ) NOT NULL,
+    name character varying NOT NULL,
+    url character varying NOT NULL
+);
+
+ALTER TABLE public."DiscoursePlatform" OWNER TO "postgres";
+
+COMMENT ON TABLE public."DiscoursePlatform" IS
+'A data platform where discourse happens';
+
+CREATE TABLE IF NOT EXISTS public."DiscourseSpace" (
+    id bigint DEFAULT nextval(
+        'public."entity_id_seq"'::regclass
+    ) NOT NULL,
+    url character varying,
+    name character varying NOT NULL,
+    discourse_platform_id bigint NOT NULL
+);
+
+ALTER TABLE public."DiscourseSpace" OWNER TO "postgres";
+
+COMMENT ON TABLE public."DiscourseSpace" IS
+'A space on a discourse platform representing a community engaged in a conversation';
+
+ALTER TABLE ONLY public."DiscoursePlatform"
+ADD CONSTRAINT "DiscoursePlatform_pkey" PRIMARY KEY (id);
+
+ALTER TABLE ONLY public."DiscourseSpace"
```

Comment:
@maparent renaming this to just `Space` (and `Platform`) instead of `DiscourseSpace` keeps nagging at me.  "Discourse` feels redundant.  Thoughts? Or is it named this way for a specific purpose (eg: to avoid a clash of some sort)?

Reply:
No anticipated conflict, and no objection. I'll do that rename later today.